data/helpers.d/utils

```diff
@@ -24,3 +24,19 @@ ynh_get_plain_key() {
         fi
     done
 }
+
+# Curl abstraction to simplify its call
+#
+# Helpfully mostly for curl arguments
+#
+# example: ynh_curl_abstract "value1=$var1&value2=$var2" "/install.php?installButton"
```

Comment:
Here one could assume that you have to pass a variable systematically, although it could be a constant.
Maybe you can reformulate: `"key1=value1&key2=value2"` and state `where you can use shell variables as values (e.g. key1=$var1)`